java/service-feature-table-manualcache/README.md

```diff
@@ -1,18 +1,39 @@
-# Service Feature Table (Manual Cache)
-Use a feature service in manual cache mode. This mode does not fetch features until a call to populate from service is called. From this point onwards, no additional queries are sent to the server to retrieve features. If you know what features you need ahead of time, this is the best mode.
+# Service feature table (manual cache)
 
-![Service FeatureTable Manual Cache App](service-feature-table-manualcache.png)
+Display a feature layer from a service using the **manual cache** feature request mode.
+
+![Image of service feature table manual cache](service-feature-table-manualcache.png)
+
+## Use case
+
+`ServiceFeatureTable` supports three request modes, which define how features are requested from the service and stored in the local table. The feature request modes have different performance characteristics. Use **manual cache** in scenarios where you want to explicitly control requests for features. See [Table performance concepts](https://developers.arcgis.com/android/latest/guide/layers.htm#ESRI_SECTION1_40F10593308A4718971C9A8F5FB9EC7D) in the *ArcGIS Runtime SDK for Android* guide to learn more.
 
 ## How to use the sample
-Simply run the app.
+
+Run the sample and pan and zoom around the map. Features will be requested and displayed automatically.
 
 ## How it works
-Set the `ServiceFeatureTable.FeatureRequestMode` to `MANUAL_CACHE` on an service feature table before it is loaded, then call populate from service using a query which defines the required features.
+
+1. Set the `ServiceFeatureTable.FeatureRequestMode` property of the service feature table to `MANUAL_CACHE` before the table is loaded.
+2. Load the table with `serviceFeatureTable.loadAsync`.
+3. Call `populateFromServiceAsync` on the table to request features.
 
 ## Relevant API
+
 * FeatureLayer
+* FeatureRequestMode.MANUAL_CACHE
 * ServiceFeatureTable
-* ServiceFeatureTable.setFeatureRequestMode(...)
+* ServiceFeatureTable.populateFromServiceAsync
+* ServiceFeatureTable.setFeatureRequestMode
+
+## About the data
+
+The sample opens with an initial visible extent centered over pool permit statuses in California.
```

Comment:
I don't think this one is pool permit statuses, as it centers on a bunch of trees.

Reply:
oh my goodness, a different data set again?! Aaaah ok thanks I'll make another description. Just assumed this would use the same as the others!